lib/plugins/interactiveCli/initializeService.js

```diff
@@ -43,8 +43,13 @@ const projectNameInput = async (workingDir) =>
             "   - Shouldn't exceed 128 characters"
           );
         }
-        if (await fse.exists(join(workingDir, input))) return `Path ${input} is already taken`;
-        return true;
+
+        try {
+          await fs.promises.access(join(workingDir, input));
+          return `Path ${input} is already taken`;
+        } catch {
+          return true;
+        }
```

Comment:
Hey @medikoo, these statements can also be written like this
`fs.promises.access(join(workingDir, input))
      .then(() => Path ${input} is already taken)
      .catch(() => true)`
Which one do you prefer?

Reply:
@sdas13 we definitely should stick to async/await syntax. It's way more readable and maintanable